CONTRIBUTING.md

```diff
@@ -33,16 +35,16 @@ Good bug reports are extremely helpful, so thanks!
 
 Guidelines for bug reports:
 
-0. Please don't report issues with games where you changed NewGRFs.
+0. Please don't report issues with games where you changed NewGRFs mid-game. (This can be verified with the `gamelog` console command in-game.)
 
-1. Please don't report issues with modified versions of OpenTTD (patchpacks and similar).
+1. Please don't report issues with modified versions of OpenTTD (patchpacks, unofficial ports, and similar).
 
-2. **Use the GitHub issue search** --- check if the issue has already been
+2. **Use the GitHub issue search** — check if the issue has already been
```

Comment:
Why this change, ooi?

Reply:
When viewed in proportional fonts, a proper em-dash looks better than three hyphens. So entirely cosmetic.